cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -132,7 +133,8 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
     testQuery(
       title = "Drop a non-existing index",
-      text = "If it is uncertain if an index exists and you want to drop it if it does but not get an error should it not, use: ",
+      text = "If it is uncertain if an index exists and you want to drop it if it does but not get an error should it not, use `IF EXISTS`. " +
+      "Note: The `IF [NOT] EXISTS` syntax for indexes is only available in Neo4j 4.1.3 and onwards.",
```

Comment:
Since this is `DROP` the `NOT` should not exist
```suggestion
      "Note: The `IF EXISTS` syntax for indexes is only available in Neo4j 4.1.3 and onwards.",
```